bin/exec.go

```diff
@@ -71,11 +58,16 @@ func main() {
 		}
 
 		experiment := utils.NewExperimentDetails()
-		experiment.Name = engineDetails.Experiments[i]
-		experiment.Namespace = engineDetails.AppNamespace
-		experiment.SvcAccount = engineDetails.SvcAccount
-		log.Infof("Printing experiment.Name: %v, experiment.Namespace : %v", experiment.Name, experiment.Namespace)
+		experiment.SetValueFromChaosEngine(engineDetails, i)
+		experiment.SetValueFromChaosExperiment(clients)
+		experiment.SetENV(engineDetails, clients)
+		log.Infof("Printing Experiment Structure: %v", experiment)
+
+		experimentStatus := utils.ExperimentStatus{}
+		experimentStatus.IntialExperimentStatus(experiment)
+		experimentStatus.InitialPatchEngine(engineDetails, clients)
 
+		log.Infof("Printing experiment.Name: %v, experiment.Namespace : %v", experiment.Name, experiment.Namespace)
 		log.Infoln("Going with the experiment Name: " + engineDetails.Experiments[i])
```

Comment:
log.Infoln("Preparing to run chaos experiment: " + engineDetails.Experiments[i])

Reply:
Done